src/zimg/colorspace/gamma.cpp

```diff
@@ -117,6 +120,26 @@ float rec_1886_inverse_eotf(float x) noexcept
 	return x < 0.0f ? 0.0f : zimg_x_powf(x, 1.0f / 2.4f);
 }
 
+float smpte_240m_eotf(float x) noexcept
+{
+	if (x < SMPTE_240M_BETA)
+		x = x * 4.0f;
+	else
+		x = SMPTE_240M_ALPHA * zimg_x_powf(x, 0.45f) - (SMPTE_240M_ALPHA - 1.0f);
+
+	return x;
+}
```

Comment:
SMPTE 240M like all other legacy gamma functions is a OETF, not an EOTF. The EOTF should be BT.1886.

Reply:
oh good point, should I just reflect that in the function name or is there something else I am missing?

Reply:
The function name should be updated, and in select_transfer_function, the OETF should only be returned if scene_referred is true. See the other gamma functions.

Reply:
Actually 240M has its own 

**4.4 Electro-optical transfer characteristic of reference reproducer**

Reply:
Attached.

[s240m.pdf](https://github.com/sekrit-twc/zimg/files/6447773/s240m.pdf)